src/engine/renderer/tr_shader.cpp

```diff
@@ -49,6 +49,9 @@ static cullType_t    implicitCullType;
 
 static char          whenTokens[ MAX_STRING_CHARS ];
 
+// DarkPlaces material compatibility
+static Cvar::Cvar<bool> r_dpMaterial("r_dpMaterial", "Enable DarkPlaces material compatibility", Cvar::NONE, false);
```

Comment:
@slipher, do you think this requires the annoying `#ifdef BUILD_ENGINE ` magic too?

Reply:
seems not